README.md

```diff
@@ -1 +1,8 @@
-# create202404
\ No newline at end of file
+# create202404
+Some description that doesnt mean anything.
+
+## Section 2
+How to enable something that need to be enabled without the need to enable it.
+
+## Section 1
+Section that should be before #section2 but for some reason is after
```

Comment:
The placement of sections within the documentation seems incorrect. `## Section 1` should logically appear before `## Section 2` as indicated by their order and naming convention.